### ISSUE TITLE
duplicate helptag "vim-lsp-settings"

### ISSUE THREAD

**Issue by pocari** [prabirshrestha/vim-lsp#808]
I use [dein](https://github.com/Shougo/dein.vim) to manage my plugins. I'm having trouble with this kind of error recently.

```
[dein] Error generating helptags:
[dein] Vim(helptags):E154: Duplicate tag "vim-lsp-settings" in file /Users/pocari/.cache/dein/.cache/init.vim/.dein/doc/vim-lsp.txt
[dein] function dein#recache_runtimepath[1]..dein#install#_recache_runtimepath[19]..<SNR>94_helptags, line 12
```

I looked into it, and it seems to be an error caused by using the same help tags in [vim-lsp-settings](https://github.com/mattn/vim-lsp-settings) and vim-lsp.
Is it possible to change the name of this tag?

**Comment by tsuyoshicho** [comment]
FYI

I sent fix PR #807 .

**Comment by tsuyoshicho** [comment]
Merged.
